packages/test-kit/src/with-feature.ts

```diff
@@ -225,7 +227,7 @@ export function withFeature(withFeatureOptions: IWithFeatureOptions = {}): WithF
         devtools = envDebugMode ? debugMode : undefined,
         slowMo,
         persist,
-        takeScreenshotsOfFailed = true,
+        takeScreenshotsOfFailed = process.env.TESTS_SCREENSHOTS === 'true',
```

Comment:
Maybe "TEST_SCREENSHOTS"

Also lets add some documentation for this environment variable